deviceModeInit.js

```diff
@@ -2,9 +2,8 @@ var rudderTracking = (function () {
   const pages = {
     "/products/": "Product Viewed",
     "/cart": "Cart Viewed",
-    "/collections/": "Product List Viewed",
+    "/collections/": "",
```

Comment:
This collections is mapped to empty string as we don't want to track `Product List Viewed ` event from here but we want to add listeners for `Product Clicked` event on the collections page

Reply:
No for `/collections/` we want the Product List viewed event along with adding listeners

Reply:
So what we are doing we triggering `Product list viewed` event not only for collection but for `/products` as well.
So the whole flow changed due which we were not adding Listners through that.

Reply:
Remove this mapping then